src/js/entities-service/entities/actions.js

```diff
@@ -117,6 +117,23 @@ const _Model = BaseModel.extend({
 
     return false;
   },
+  canSubmit() {
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+
+    if (currentUser.can('work:submit')) return true;
+
+    if (currentUser.can('work:owned:submit') && this.getOwner() === currentUser) return true;
+
+    if (currentUser.can('work:team:submit')) {
+      const owner = this.getOwner();
+      const currentUsersTeam = currentUser.getTeam();
+      const ownersTeam = owner.type === 'teams' ? owner : owner.getTeam();
+
+      if (currentUsersTeam === ownersTeam) return true;
+    }
+
+    return false;
+  },
```

Comment:
My assumption is that we want this to be independent of the `work:manage` (`canEdit()`) permissions.

Because of that assumption, I did not include a `if (!this.canEdit()) return false;` check at the top of this function.